src/app/pages/ParaTimesPage/useParaTimes.ts

```diff
@@ -94,7 +94,7 @@ export const useParaTimes = (): ParaTimesHook => {
     accountAddress,
     accountIsLoading,
     availableParaTimesForSelectedNetwork,
-    balance: walletBalance,
+    balance: walletBalance || '0',
```

Comment:
Related: ParaTimesState.balance type should be `| null` instead of
```ts
export interface ParaTimesState {
  balance: StringifiedBigInt

export const initialState: ParaTimesState = {
  balance: '',
```